src/werkzeug/serving.py

```diff
@@ -103,7 +103,7 @@ def readinto(self, buf):
                 # buffer. If this operation fully consumes the chunk, this will
                 # reset self._len to 0.
                 n = min(len(buf), self._len)
-                buf[read : read + n] = self._rfile.read(n)
+                buf[read: read + n] = self._rfile.read(n)
```

Comment:
Please do not do unrelated formatting changes in PRs.

Reply:
Sorry – I think my editor has "format on save" set by default.